src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -220,4 +268,4 @@ class ActivityFeed extends React.Component<Props, State> {
     }
 }
 
-export default ActivityFeed;
+export default withLoggerContext(ActivityFeed);
```

Comment:
Why wrap the consumer with a HOC but use the context API directly for `ContentSidebar.js`?